.github/workflows/release.yaml

```
@@ -39,6 +39,26 @@ jobs:
           assets: |
             release.zip
 
+      # The GitHub list-releases API is eventually consistent, so the draft
+      # created above is sometimes not visible yet when publish runs, which
+      # makes GitReleaseManager crash with a NullReferenceException.
+      - name: Wait for draft release to be visible
+        env:
+          GH_TOKEN: ${{ secrets.PERSONAL_GITHUB_TOKEN }}
+          TAG: ${{ steps.extract_version.outputs.version }}
```

actionlint (1.7.12)

[error] property "extract_version" is not defined in object type {}

(expression)